test/framework/options.py

```diff
@@ -3570,13 +3479,7 @@ def test_force_download(self):
             '--force-download',
             '--sourcepath=%s' % self.test_prefix,
         ]
-        self.mock_stdout(True)
-        self.mock_stderr(True)
-        self.eb_main(args, raise_error=True, verbose=True, do_build=True)
-        stdout = self.get_stdout().strip()
-        stderr = self.get_stderr().strip()
-        self.mock_stdout(False)
-        self.mock_stderr(False)
+        stdout, stderr = self._run_mock_eb(args, do_build=True, raise_error=True, verbose=True, strip=True)
         self.assertEqual(stdout, '')
```

Comment:
@boegel this is failing, but I don't understand, why should `stdout` be empty?

It doesn't fail without the new helper method, so I must be missing something...